packages/cli/src/commands/index.ts

```diff
@@ -1,7 +1,9 @@
 import {Command, DetachedCommand} from '@react-native-community/cli-types';
-import start from './start/start';
-import bundle from './bundle/bundle';
-import ramBundle from './bundle/ramBundle';
+import {
+  startCommand as start,
+  bundleCommand as bundle,
+  ramBundleCommand as ramBundle,
+} from '@react-native-community/cli-plugin-metro';
```

Comment:
In the future, we can remove this when this plugin becomes direct React Native dependency. 

Here's a snippet:
https://github.com/facebook/react-native/blob/main/react-native.config.js#L16

I believe it should simply say `metro.commands`, but this is to be decided on the planning meeting.